packages/free/src/adapter.ts

```diff
@@ -77,6 +77,10 @@ export function freeStorage<T>(token: string, opts?: StorageOptions) {
   const storage = new Storage(token, opts?.rootUrl);
   if (opts?.jwt !== undefined) storage.jwt = opts.jwt;
   return {
+    async readKeys(): Promise<T> {
```

Comment:
Let's adhere to the interface we implement.
```suggestion
    async readAllKeys(): Promise<T> {
```
Spec: https://grammy.dev/ref/core/storageadapter#readallkeys

Reply:
Yes, much more readable.